src/routes/github/github-router.ts

```diff
@@ -7,27 +7,35 @@ import { GithubConfigurationRouter } from "routes/github/configuration/github-co
 import { returnOnValidationError } from "../api/api-utils";
 import { header } from "express-validator";
 import { WebhookReceiverPost } from "./webhook/webhook-receiver-post";
+import { GithubServerAppMiddleware } from "middleware/github-server-app-middleware";
 
 export const GithubRouter = Router();
 
+const GithubAppProvidedRouter = Router({ mergeParams: true });
+GithubRouter.use("/:appId?", GithubAppProvidedRouter);
```

Comment:
With this optional path params, when test the cloud scenario, express will match the `:appId` with `/configuration` from the original url `/github/configuration`.

But since it is matching the appId with 'configuration', the 'configuration' is remove from the `req.path`, therefore it wont' match the following `GitHubConfigurationGet` route.

![image](https://user-images.githubusercontent.com/105693507/179154706-85a4bdec-c3bd-4626-83cd-04fce93bb284.png)

Reply:
Added a commit with an example of how it should work.  I didn't test it at all.